contracts/Stake/StakeManager.sol

```diff
@@ -13,28 +13,22 @@ contract StakeManager is Initializable, Ownable {
 
     using SafeMath for uint256;
 
-    struct TokenData {
-        uint256 amount;
-        uint256 index; // index in tokensDelegated array
-    }
-
     struct Stash {
         address staker;
         address delegatedCluster;
-        mapping(bytes32 => TokenData) amount;   // name is not intuitive
+        mapping(bytes32 => uint256) amount;   // name is not intuitive
```

Comment:
We probably can't do this because current storage will be effected. I mean previously  store values cannot be retrieved.